src/tox/plugin/manager.py

```diff
@@ -15,6 +15,7 @@
 from tox.tox_env.python.virtual_env.package import api
 from tox.tox_env.register import REGISTER, ToxEnvRegister
 
+from ..config.main import Config
```

Comment:
Why not `from tox.config.main import Config`?

Reply:
Prefer relative imports where possible :thinking: